webapp/src/extensionsBrowser.tsx

```diff
@@ -33,6 +33,7 @@ export const ExtensionsBrowser = (props: ExtensionsProps) => {
     const [searchFor, setSearchFor] = useState("");
     const [searchComplete, setSearchComplete] = useState(true)
     const [allExtensions, setAllExtensions] = useState(fetchBundled());
+    const [extensionsInDevelopment, _] = useState(fetchLocalRepositories());
```

Comment:
nit: you can use `useMemo(() => fetchLocalRepositories(), [])`

Reply:
Is this syntactic sugar for what I've done here? Or does it work in a fundamentally different way?

Reply:
This uses `useMemo` now.